Doc/library/os.rst

```diff
@@ -4654,6 +4654,9 @@ written in Python, such as a mail server's external command delivery program.
 
    .. availability:: Unix, not Emscripten, not WASI.
 
+   .. note::
+      This function is not available on MacOS.
```

Comment:
```suggestion
      This function is not available on macOS.
```

IMO, it spelles as `macOS` in the rest of this document.

Reply:
Can we just change the exist `availability` section up here?

Reply:
@aisk: On spelling, the document is not consistent and it appears both ways. As I noted in the description above:
> I've followed the model of os.fdatasync()

and that is both as to the spelling/casing ("MacOS" vs. "macOS") as well as the Note section rather than the Availability section. It is not how I would have written it without that precedent.

That said, I'm happy to conform to whatever standard, but the document style is inconsistent, and I'd really just like the semantics fixed before the next release, I don't care about the style. I'd tend to think the style correction should be a different PR that works harder to make everything consistent, but I'm not involved enough in Python development to know the proper cultural processes.

Reply:
The correct spelling is "macOS" according to Apple. I'll fix the spelling in the `fdatasync` documentation in a follow-up PR.

Documenting this in a note is fine, as you note we do this in more places.